Source/PeleLMeX_Utils.cpp

```diff
@@ -2201,20 +2201,18 @@ PeleLM::extendSignedDistance(
   // First set the region far away at the max value we need
   const auto& dx = geom[0].CellSizeArray();
```

Comment:
Sorry I messed up the conflict resolution. This line should replace line 2205 I believe.